.appveyor.yml

```diff
@@ -64,8 +64,7 @@ install:
           Add-Content php.ini "`n extension=php_sqlite3.dll"
           Add-Content php.ini "`n extension=php_curl.dll"
 
-          # Get and install the latest stable sqlsrv DLL's
-          $DLLVersion = (Invoke-WebRequest "https://pecl.php.net/rest/r/sqlsrv/stable.txt").Content
+          $DLLVersion = "5.10.0"
```

Comment:
Windows builds are not always up to speed with PECL. It's been working so far thanks to the AppVeyor cache. We can switch back to the latest stable later once they sync up.